Order creation API endpoints updated for guest users (#6086)
magento/devdocs: src/guides/v2.2/rest/tutorials/orders/order-add-items.md

This article shows how to add a simple product, a downloadable product, and a [bundle product](https://glossary.magento.com/bundle-product) to the cart.

> This calls are performed on behalf of a customer, and the customer's token is specified in the [authorization](https://glossary.magento.com/authorization) header.

Reviewer: Please, adjust

```suggestion
These calls are performed on behalf of a customer, and the customer's token is specified in the [authorization](https://glossary.magento.com/authorization) header.
```

Author: Updated as per your suggestion

{% endcollapsible %}

> This calls are performed on behalf of a guest-customer, and there is no need of token.

Reviewer: Please, adjust
```suggestion
These calls are performed on behalf of a guest-customer, and there is no need for a token.
```

Author: Updated as per your comments

> ### Add a simple product to a cart {#add-simple}

Reviewer: I believe we need to clarify here that we are talking about a guest user. Otherwise, we have two identical topics (and identical anchors, this point needs to be adjusted as well).

Author: Updated as per your comments

**Endpoint:**

`POST <host>/rest/<store_code>/guest-carts/<cartId>/items`

> Cart Id is the quoteId recieved on quote creation.

Reviewer: Could I kindly ask you to make the [quote creation] as a link to the corresponding topic? Thank you

Author: Updated as per your comments

Author: Updated as per your comments

**Headers:**

`Content-Type` `application/json`

**Payload:**

```json
{
  "cartItem": {
    "sku": "WS12-M-Orange",
    "qty": 1,
    "quote_id": "5JfSLCdQUP4gwmM4z8u74iuPa0kfkZ5l"
  }
}
```
**Response:**

```json
{
  "item_id": 7,
  "sku": "WS12-M-Orange",
  "qty": 1,
  "name": "Radiant Tee-M-Orange",
  "product_type": "simple",
  "quote_id": "27"
}
```

> Note: The payload and response is same for all products apart from quote id in the payload (Need to give the masked ID)

Reviewer: Please adjust

```suggestion
Note: The payload and response are the same for all products apart from quote id in the payload (Need to give the masked ID)
```

Author: Updated as per your comments
